styles.css

```diff
@@ -114,3 +114,417 @@ p {
+  color: #fff;
+  border: #fff;
+}
+
+.details {
+  background-image: url(images/bg-img.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 100%;
+  color: white;
+  padding-top: 20%;
+  margin: 35px;
+}
+
+/* about me section */
+
+#about-me-sec {
+  background-color: #1c1a19;
+  color: #fff;
+  align-items: center;
+  text-align: center;
+}
+
+.about-me {
+  background-image: url(images/about-me.png);
+  background-repeat: no-repeat;
+  background-position: bottom right;
+  padding-right: 24px;
+  padding-bottom: 220px;
+  align-items: flex-start;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: left;
+}
+
+.get-my-resume-btn {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  background: #ff6b00;
+  border: #fff;
+  text-align: left;
+  padding: 12px;
+  padding-left: 20px;
+  margin-left: 25px;
+}
+
+#color {
+  margin-left: 15px;
+  color: #091e42;
+}
+
+.vertical-list,
+.stack-Skills {
+  align-items: center;
+  text-align: center;
+}
+
+/* .about-me {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: left;
+} */
+
+.about-header-txt {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 88px;
+  color: #ff6b00;
+  padding: 30px 0 0 20px;
+}
+
+.about-txt {
+  padding: 30px 0 0 20px;
+  padding-bottom: 20px;
+  margin-bottom: 20px;
+}
+
+.vert-title {
+  font-family: 'InaiMathi', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+/* .lang-stack {
+    font-family: 'Inter';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 43px;
+    text-align: center;
+    align-items: center;
+    padding-left: 120px;
+
+}
+.frame {
+    font-family: 'Inter';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 43px;
+    text-align: center;
+    align-items: center;
+    padding-left: 25px;
+}
+.skill-list {
+    font-family: 'Inter';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 43px;
+    text-align: center;
+    align-items: center;
+    padding-left: 25px; */
+
+.stack-items {
+  align-items: center;
+}
+
+.stack-items li {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 43px;
+  text-align: center;
+  align-items: center;
+  justify-content: center;
+
+  /* padding-left: 120px; */
+}
+
+.stack-title {
+  display: flex;
+  align-items: center;
+  font-family: 'InaiMathi', sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  justify-content: center;
+}
+
+.list li {
+  margin: 32px;
+  list-style: none;
+}
```

Comment:
- [ ] kindly consider removing the scroll on the x-axis for the best user experience check on the below screenshoot you will see that its being scrolled left and light 
<img width="418" alt="Screen Shot 2023-03-23 at 18 30 07" src="https://user-images.githubusercontent.com/60146030/227271883-99e4eb87-4dfe-4872-840a-b34f70f7ae86.png">

Reply:
hi [divinecharlotte](https://github.com/divinecharlotte)
kindly help with this it got me stuck i cant figure out please kindly sugest how to

Reply:
how to remove the scroll

Reply:
> how to remove the scroll

Hello @techmoves you can remove it by making it mobile responsive

Reply:
hi [braincee](https://github.com/braincee) 

can you please kindly check the adjustment i did please